Marlin/Configuration_adv.h

```diff
@@ -1044,7 +1045,7 @@
 #define I2C_SLAVE_ADDRESS  0 // Set a value from 8 to 127 to act as a slave
 
 /**
- * Add M43, M44 and M45 commands for pins info and testing
```

Comment:
M44 & M45 are utilities that Roxy uses.  One of them is similar to the W option in M43.  The other is the Toggle function being added on this PR.